pkg/pillar/devicenetwork/wpad.go

```diff
@@ -61,7 +66,7 @@ func CheckAndGetNetworkProxy(log *base.LogObject, portStatus *types.NetworkPortS
 	// in DomainName until we succeed
 	for {
 		url := fmt.Sprintf("http://wpad.%s/wpad.dat", dn)
```

Comment:
I can see that we use `"WpadURL":"http://wpad.sdn./wpad.dat"` (in DeviceNetworkStatus) if we see domain/search `sdn` in `resolv.conf`:
```
712fc5e7-4d16-4bd8-a7f8-f1fbaf5488c2:/persist# cat /run/dhcpcd/resolv.conf/eth0.dhcp 
# Generated by dhcpcd from eth0.dhcp
domain sdn
search sdn
nameserver 10.16.16.25
```

Should we avoid usage of dot in domain name here?

Reply:
Actually the trailing dot is correct, even if it is typically omitted: http://www.dns-sd.org/trailingdotsindomainnames.html